example-external-task-worker/src/main/java/info/novatec/micronaut/camunda/externaltask/worker/ExternalTaskWorker.java

```diff
@@ -34,13 +34,13 @@ void onStartup(ServerStartupEvent event) {
         log.info("Starting ExternalTaskClient");
 
         ExternalTaskClient client = ExternalTaskClient.create()
-                .baseUrl("http://localhost:8080")
+                .baseUrl("http://localhost:8080/engine-rest")
                 .asyncResponseTimeout(10000) // long polling timeout
                 .build();
 
 
         // subscribe to an external task topic as specified in the process
-        client.subscribe("item")
+        client.subscribe("my-item")
```

Comment:
this should be "my-topic".
Please fix this - do a final test and then we can merge it.

Reply:
re-opening. It still says "my-item" here instead of "my-topic".